packages/front-end/services/datasources.ts

```diff
@@ -979,6 +1036,12 @@ export function getInitialDatasourceResources({
       userIdTypes: datasource.settings?.userIdTypes || [],
     });
 
+  if (datasource.type === "clickhouse")
```

Comment:
The type should be "growthbook-clickhouse" as other users using clickhouse are not going to have the same schema.

Reply:
Also an old diff!